store/src/main/java/RandomStorePopulator.java

```diff
@@ -0,0 +1,33 @@
+import com.github.javafaker.Faker;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.ArrayList;
+
+public class RandomStorePopulator {
+
+    public ArrayList<Product> populateTheStore() {
+
+        Faker faker = new Faker();
+        ArrayList<Product> products = new ArrayList<>();
+        for (int i = 0; i <= 15; i++) {
+
+            Category myCategory = null;
```

Comment:
so, all this code is not what was required. The task is to read all Category class implementations in some package, and create their instances and populate some products to them